src/main/java/com/autotune/common/auth/AuthenticationStrategyFactory.java

```diff
@@ -0,0 +1,29 @@
+package com.autotune.common.auth;
+
+import com.autotune.utils.KruizeConstants;
+
+public class AuthenticationStrategyFactory {
+
+    public static AuthenticationStrategy createAuthenticationStrategy(AuthenticationConfig authConfig) {
```

Comment:
Also support for no auth stratgey

Reply:
Support for no auth has been added in the AuthenticationConfig class:

https://github.com/khansaad/autotune/blob/f0c5b16d9759c62a04feff09597a7d9ce99e4ec7/src/main/java/com/autotune/common/auth/AuthenticationConfig.java#L64